samples/matter/common/src/dfu/smp/dfu_over_smp.cpp

```diff
@@ -16,43 +17,42 @@
 
 #include <lib/support/logging/CHIPLogging.h>
 
+#ifndef CONFIG_SUIT
 #include <zephyr/dfu/mcuboot.h>
+#include <zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt.h>
+#endif
+
 #include <zephyr/mgmt/mcumgr/mgmt/callbacks.h>
 #include <zephyr/mgmt/mcumgr/mgmt/mgmt.h>
-#include <zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt.h>
 
 using namespace ::chip;
 using namespace ::chip::DeviceLayer;
 
-constexpr uint8_t kAdvertisingPriority = UINT8_MAX;
-constexpr uint32_t kAdvertisingOptions = BT_LE_ADV_OPT_CONNECTABLE;
-constexpr uint16_t kAdvertisingIntervalMin = 400;
-constexpr uint16_t kAdvertisingIntervalMax = 500;
-constexpr uint8_t kAdvertisingFlags = BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR;
+constexpr static uint8_t kAdvertisingPriority = UINT8_MAX;
+constexpr static uint32_t kAdvertisingOptions = BT_LE_ADV_OPT_CONNECTABLE;
+constexpr static uint16_t kAdvertisingIntervalMin = 400;
+constexpr static uint16_t kAdvertisingIntervalMax = 500;
+constexpr static uint8_t kAdvertisingFlags = BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR;
 
 namespace
 {
-enum mgmt_cb_return UploadConfirmHandler(uint32_t event,
-					 enum mgmt_cb_return prev_status,
-					 int32_t *rc, uint16_t *group,
-					 bool *abort_more, void *data,
-					 size_t data_size)
+enum mgmt_cb_return UploadConfirmHandler(uint32_t event, enum mgmt_cb_return prev_status, int32_t *rc, uint16_t *group,
+					 bool *abort_more, void *data, size_t data_size)
 {
+/* Currently img_mgmt hooks are not supported by SUIT */
```

Comment:
Are we able to track DFU progress some other way? If not, could we somehow inform the user that DFU started and he should wait some time for it to end? I'm afraid that customer can be worried that application stuck, as progress will not be shown.

Reply:
Not necessarily, it's ok, as long as we track this.